README.md

```diff
@@ -21,12 +21,3 @@ go vet -vettool=$(which sqlclosecheck) ./...
 go install github.com/ryanrolds/sqlclosecheck@latest
 go vet -vettool=${GOPATH}/bin/sqlclosecheck ./...
 ```
-
-## Roadmap
-
-* ~~Get linter working~~
-* ~~Added some basic test cases~~
-* ~~Require that Close be deferred~~
-* ~~Add sqlx checking~~
-* ~~Test across a bunch of projects~~
-* ~~Introduce linter to [golangci-lint](https://github.com/golangci/golangci-lint).~~
```

Comment:
Just some cleanup while I was here. I feel that a finished roadmap doesn't add much value to the README but let me know if you want to keep it.